tests/src/Kernel/Controller/PackageControllerKernelTest.php

```diff
@@ -54,12 +55,17 @@ protected function setUp() {
     $this->installConfig([
       'user',
     ]);
+    // Install user 0 and user 1.
+    \user_install();
```

Comment:
Maybe remove `slash` at the beginning of the function name ? Is it really needed ?